### PR TITLE
UI: Add handling for empty state flaky test edge case

### PR DESCRIPTION
**Pull request by backspace**
This test occasionally fails because the random generation
sometimes produces a node with no allocations, such as [here](https://app.circleci.com/pipelines/github/hashicorp/nomad/12735/workflows/9a407f87-0383-48ee-8e41-7e6e351dce28/jobs/113130/steps).

This is one way to solve it without trying to force the generated
data to always have an allocation.

### CODE REVIEW

File: ui/tests/acceptance/topology-test.js [hunk 1]
```diff
@@ -56,8 +56,12 @@ module('Acceptance | topology', function(hooks) {
 
     await Topology.visit();
 
-    await Topology.viz.datacenters[0].nodes[0].memoryRects[0].select();
-    assert.equal(Topology.infoPanelTitle, 'Allocation Details');
+    if (Topology.viz.datacenters[0].nodes[0].isEmpty) {
+      assert.expect(0);
```
Comment (DingoEatingFuzz) [inline ui/tests/acceptance/topology-test.js:60]: I don't like that whether or not this test has assertions is at the whim of a faker seed, but I won't make you fix that. I'll get to it with my ongoing to top viz stuff.

**Review by DingoEatingFuzz — APPROVED** [review]
Thank you for addressing this. As I mention, I'd prefer the fixture data be changed to always exercise the behavior, but I'll fix that.

This fixes the immediate issue of the flaky test, so I'm happy with this.

**Comment by backspace** [comment]
agreed re 0 assertions, I hope you can figure out a way around it 🤞
